geomesa-accumulo/geomesa-accumulo-datastore/src/main/scala/org/locationtech/geomesa/accumulo/stats/StatWriter.scala

```diff
@@ -157,6 +157,7 @@ object StatWriter extends Runnable with Logging {
       // get the appropriate transform for this type of stat
       val transform = group.clas match {
         case c if c == classOf[QueryStat] => QueryStatTransform.asInstanceOf[StatTransform[Stat]]
+        case r if r == classOf[RasterQueryStat] => RasterQueryStatTransform.asInstanceOf[StatTransform[Stat]]
```

Comment:
[picky][notyourfault] Try to optimize imports on this file....it looks like there are two unused imports